package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-cli-plugin-asset-compute",
-  "description": "Asset Compute Plugin for Adobe I/O Command Line Interface",
+  "description": "Tool to develop and test Adobe Asset Compute workers",
   "version": "1.0.3",
```

Comment:
I would up minor, as standalone is a new feature (at least the way I see it).

Reply:
This description from the package is used by oclif for the help output (in standalone mode), hence I changed this to something that makes sense there. See output in https://github.com/adobe/aio-cli-plugin-asset-compute/pull/17#issuecomment-638546959.